.env.example

```diff
@@ -23,11 +23,16 @@ DIRECT_URL="postgresql://postgres:${POSTGRES_PASSWORD}@localhost:${POSTGRES_PORT
 NEXTAUTH_SECRET="CvN2qK1adeaBxgcgTmnMNzGPO4SPrs2yuF1RVfQ/VY4=" # not used for production
 NEXTAUTH_URL="http://localhost:3000"
 
+# Posthog
+# You can generate your own posthog project at https://posthog.com/
+NEXT_PUBLIC_POSTHOG_KEY="YOUR POSTHOG KEY HERE"
+NEXT_PUBLIC_POSTHOG_HOST=https://us.i.posthog.com
+
 # Supabase
 # https://supabase.io/docs/reference/environment-variables
 NEXT_PUBLIC_SUPABASE_URL="https://PROJECT_ID.supabase.co"
 NEXT_PUBLIC_SUPABASE_ANON_KEY="SUPABASE_ANON_KEY"
 SUPABASE_SERVICE_ROLE_KEY="SUPABASE_SERVICE_ROLE_KEY"
 
 # School domains (comma-separated)
-NEXT_PUBLIC_SUPPORTED_SCH_DOMAINS="sch1.com,sch2.com"
+NEXT_PUBLIC_SUPPORTED_SCH_DOMAINS="sch1.com,sch2.com"
```

Comment:
Change request: This change seems to be caused by additional blank like at the bottom of the file, will recommend to remove since it is not related to this PR

Reply:
does not seem to be the case, there is no blank line